gallery/visualtests/nxDrawer.js

```diff
@@ -125,6 +125,48 @@ describe('NxDrawer', function() {
     });
   });
 
+  describe('NxDrawer with NxToast', function() {
+    const browserSetup = setupBrowser('#/NxDrawerWithNxToastExample', false);
+    const { getPage, checkFullPageScreenshot, waitAndGetElements, wait, isVisible } = browserSetup;
+    const openDrawer = openDrawerFromPage(browserSetup);
+
+    const buttonId = 'nx-drawer-with-nx-toast-open-button';
+    const drawerId = 'nx-drawer-with-nx-toast';
+
+    it('is positioned correctly', async function() {
+      await getPage().setViewport(viewportSize);
+      const [openSuccessToastButton] = await waitAndGetElements('#nx-toast-success-open-button');
+      const [openErrorToastButton] = await waitAndGetElements('#nx-toast-error-open-button');
```

Comment:
The reason `waitAndGetElements` returns an array is because it can take multiple arguments and fetch multiple `ElementHandle`s at the same time.  This reduces the round trips between the test and the browser (which is a separate process) and speeds up the tests

Reply:
> The reason `waitAndGetElements` returns an array is because it can take multiple arguments and fetch multiple `ElementHandle`s at the same time. This reduces the round trips between the test and the browser (which is a separate process) and speeds up the tests

Thanks for letting me know that can be done! : ec1c68211bc63b6f64180fee56d855cd79f3042f